ehcache/test/play/cache/EhCacheImplTest.java

```diff
@@ -63,7 +63,7 @@ public void stop() {
       fail("must throw exception");
     } catch (IllegalStateException ignored) {
     } finally {
-      EhCacheImpl.newInstance();
+      EhCacheImpl.testInstance();
```

Comment:
Why this line is needed? It doesn't have any effect.

Reply:
@asolntsev I missed this comment. You are right, this can be removed... Will do so in a bit.

Reply:
ah yes, now I see that `EhCacheImpl` constructor is private. Then it makes sense. 
@cies But I would remove `public` modifier from `EhCacheImpl.testInstance` method. Then it will be _package private_ - this is the usual practice for methods "made available only for tests".